src/app/examples/si-text-formatters/si-markdown-content.ts

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) Siemens 2016 - 2025
+ * SPDX-License-Identifier: MIT
+ */
+import { Component } from '@angular/core';
+import { SiMarkdownContentComponent } from '@siemens/element-ng/text-formatters';
+
+@Component({
+  selector: 'app-sample',
+  imports: [SiMarkdownContentComponent],
+  templateUrl: './si-markdown-content.html'
+})
+export class SampleComponent {
+  markdownContent = `# AI Assistant Response
```

Comment:
instead of inlining the markdown text, wouldn't it be nicer to just load it as example-text.md file?